frontend/dialogs/OBSAbout.cpp

```diff
@@ -1,14 +1,18 @@
 #include "OBSAbout.hpp"
-
-#include <widgets/OBSBasic.hpp>
+#include <OBSApp.hpp>
 #include <utility/RemoteTextThread.hpp>
+#include <utility/platform.hpp>
 
 #include <qt-wrappers.hpp>
 
 #include <json11.hpp>
 
 #include "moc_OBSAbout.cpp"
 
+namespace {
+QString patronJson;
+} // namespace
```

Comment:
Personally I disagree with this solution. Now the ownership of this information is no longer in OBSBasic (where yes, it doesn't really belong), but now it's owned by the compilation unit. This muddies the ownership a bit.
In my opinion, this should be managed by OBS *somewhere*. Possibly OBSApp.